extensions/interactions/MultipleChoiceInput/protractor.js

```diff
@@ -36,14 +36,20 @@ var customizeInteraction = function(elem, richTextInstructionsArray) {
 // the options.
 var expectInteractionDetailsToMatch = function(
     elem, richTextInstructionsArray) {
-  elem.all(by.repeater('choice in $ctrl.choices track by $index'))
+  elem.all(by.repeater('choice in $ctrl.choices'))
```

Comment:
Adding @U8NWXD to check whether this can cause flakiness? Considering we have recently resolved one e2e test issue which was around selecting elements using the `ng-if` attribute.

Reply:
I haven't seen any examples of flakes caused by this kind of selector, but it's not as readable as HTML classes, especially for new contributors. @rohitkatlaa is it possible to add an HTML class to each of the repeated elements and use that as the selector for `element.all(`?

Also /cc @nithusha21 in case he knows of any flakes caused by this kind of code

Reply:
@U8NWXD I have added a html class. PTAL.

Reply:
(For ref: I was pointing to [this change](https://github.com/oppia/oppia/pull/9068/files#diff-1031f0f30ce6a90c24332497a575023cL35).)

Reply:
@rohitkatlaa looks good! Thanks